cmd/lakectl/cmd/tag.go

```diff
@@ -78,8 +78,8 @@ var tagCreateCmd = &cobra.Command{
 		force, _ := cmd.Flags().GetBool("force")
 		if force {
 			// checking validity of the commitRef before deleting the old one
-			resp, err := client.GetCommitWithResponse(ctx, tagURI.Repository, commitRef)
-			DieOnErrorOrUnexpectedStatusCode(resp, err, http.StatusOK)
+			res, err := client.GetCommitWithResponse(ctx, tagURI.Repository, commitRef)
```

Comment:
Sorry for being unclear: I really liked this change, it's part of the res -> resp fix.  It caused an issue with this PR, so I think we have no choice but to make this unrelated change here.  (Technically we could have a preview PR that just fixed this, and then your PR.  Too much effort for too little gain probably.)

Reply:
not sure how we can fix this in this case. We are trying to assign `resp` in l.84 with a different type. Maybe change `res` here to be `_resp`?